gui/README.md

```diff
@@ -8,30 +8,41 @@ npm install
 ### To run the GUI with Cassy
 See the [Getting Started](../docs/getting-started.md) to set up your Cassy configuration and run the server.
 
-Once you have started Cassy, go to the root folder and run the following command to start the HTTP/1.1 server.
+Once you have started Cassy, go to the root folder and run the following commands to build and start the HTTP/1.1 server.
 ```
-./entrypoint -mode=dev
+go build entrypoint.go
 ```
-Next, start the development server by navigating to the [gui](../gui) folder and executing the following command.
 ```
-npm run serve
+./entrypoint
 ```
-If you have followed the steps correctly, you should have
-1) A running `cassy-server`
-2) A running `entrypoint` on localhost:8090
-3) A running dev server on localhost:8080
 
 In your browser, you can go to [localhost:8080](http://localhost:8080) to view the gui.
```

Comment:
I think it is `localhost`  when you run this in local so it needs to be updated ?

Reply:
Sorry, could you explain more what you mean?

Reply:
I meant, does it depend on where you run Cassy ?

Reply:
The GUI will be served on `localhost` no matter where you run `cassy-server`. I haven't tried running `entrypoint` with Cassy running at a different IP address, but it shouldn't affect the GUI as far as I understand.